src/components/sykmeldingpanel/Egenmeldingsdager.tsx

```diff
@@ -17,13 +17,10 @@ function Egenmeldingsdager({ egenmeldingsdager }: Props): ReactElement {
         <>
             <EgenmeldingsdagerList egenmeldingsdager={egenmeldingsdager} />
             <li>
```

Comment:
```suggestion
            <li className="print:hidden">
```
Kunne ha flytta `print:hidden` frå Alert til <li>. Ser det er eit litt større mellomrom under Egenmeldingsdager i forhold dei andre seksjonane.